src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -512,6 +512,111 @@ public static String composePagedCallableMethodHeaderSampleCode(
             .build());
   }
 
+  public static String composeRegularCallableMethodHeaderSampleCode(
+      Method method,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames,
+      Map<String, Message> messageTypes) {
+    VariableExpr clientVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder()
+                .setName(JavaStyle.toLowerCamelCase(clientType.reference().name()))
+                .setType(clientType)
+                .build());
+
+    // Assign method's request variable with the default value.
+    VariableExpr requestVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("request").setType(method.inputType()).build());
+    Message requestMessage = messageTypes.get(method.inputType().reference().simpleName());
+    Preconditions.checkNotNull(
+        requestMessage,
+        String.format(
+            "The request `%s` is not defined as message type.",
+            method.inputType().reference().simpleName()));
+    Expr requestBuilderExpr =
+        DefaultValueComposer.createSimpleMessageBuilderExpr(
+            requestMessage, resourceNames, messageTypes);
+    AssignmentExpr requestAssignmentExpr =
+        AssignmentExpr.builder()
+            .setVariableExpr(requestVarExpr.toBuilder().setIsDecl(true).build())
+            .setValueExpr(requestBuilderExpr)
+            .build();
+
+    List<Expr> bodyExprs = new ArrayList<>();
+    bodyExprs.add(requestAssignmentExpr);
+
+    List<Statement> bodyStatements = new ArrayList<>();
+
+    if (!method.isPaged() && !method.hasLro()) {
```

Comment:
Since this block is so long, can we add an early return (with an extra SampleCodeWriter invocation) here?

Reply:
Try to align with you. Is that mean that we could create a helper function like `composeRegularUnaryCallableBodyStatements` here?